wom/src/main/scala-2.12/wom/executable/ExecutableValidation.scala

```diff
@@ -1,29 +0,0 @@
-package wom.executable
-
-import common.Checked
-import common.validation.ErrorOr.ErrorOr
-import common.validation.Checked._
-import wom.callable.ExecutableCallable
-import wom.executable.Executable.{DelayedCoercionFunction, InputParsingFunction, ResolvedExecutableInputs}
-import wom.graph.Graph
-
-// WARNING! Because of 2.11 vs 2.12 incompatibilities, there are two versions of this file.
-// If you're making changes here, you'll also need to update ../../../scala-2.11/wom/executable/ExecutableValidation.scala
-// (More info below on why this was necessary)
```

Comment:
This warning is delete here, but the equivalent comment still remains in the newly merged copy.